builder/amazon/instance/builder.go

```diff
@@ -50,6 +50,12 @@ type Builder struct {
 }
 
 func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
+	configs := make([]interface{}, len(raws)+1)
+	configs[0] = map[string]interface{}{
+		"bundle_prefix": "image-{{timestamp}}",
+	}
+	copy(configs[1:], raws)
```

Comment:
Is this position-sensitive? I.e. could it be done as `raws = append(raws, bundle_prefix)`?